shell/platform/linux/fl_engine.cc

```diff
@@ -102,10 +153,22 @@ gboolean fl_engine_start(FlEngine* self, GError** error) {
   config.open_gl.fbo_callback = fl_engine_gl_fbo_callback;
   config.open_gl.present = fl_engine_gl_present;
 
+  FlutterTaskRunnerDescription platform_task_runner = {};
+  platform_task_runner.struct_size = sizeof(FlutterTaskRunnerDescription);
+  platform_task_runner.user_data = self;
+  platform_task_runner.runs_task_on_current_thread_callback =
+      fl_engine_runs_task_on_current_thread;
+  platform_task_runner.post_task_callback = fl_engine_post_task_callback;
```

Comment:
Nit: Not that it matters since this engine instance only has a single custom task runner, but I would set the `identifier` to something that is not default in case more task runners need interop.